tests/subsys/dfu/mcuboot/nrf52840_pca10056.overlay

```diff
@@ -20,19 +20,19 @@
 		};
 		slot0_partition: partition@c000 {
 			label = "image-0";
-			reg = <0x0000E000 0x000068000>;
+			reg = <0x0000E000 0x000066000>;
 		};
-		slot1_partition: partition@75000 {
+		slot1_partition: partition@73000 {
 			label = "image-1";
-			reg = <0x00076000 0x000068000>;
+			reg = <0x00074000 0x000066000>;
 		};
-		scratch_partition: partition@de000 {
```

Comment:
NOTE: for an overlay, the scratch and storage partitions don't change and can be removed.  But perhaps this should be in a separate PR.

Reply:
In this test storage partition size is not important - do not modify it then.
Remove this file changes form this PR.

Reply:
@nvlsianpu I can't remove all of the overlay changes as we're moving the slot1 and scratch locations.  IE:
`slot1_partition: partition@75000 {` -> `slot1_partition: partition@73000 {`
and
`scratch_partition: partition@de000 {` -> `scratch_partition: partition@da000 {`

For the .overlay to work correctly, it still needs to match on those portions.
I will remove the storage_partition portion of the overlay.

Reply:
right